tensorflow/lite/micro/python/interpreter/src/interpreter_wrapper.cc

```diff
@@ -203,7 +203,7 @@ InterpreterWrapper::~InterpreterWrapper() {
 
 InterpreterWrapper::InterpreterWrapper(
     PyObject* model_data, const std::vector<std::string>& registerers_by_name,
-    size_t arena_size) {
+    size_t arena_size, const int num_resource_variables) {
```

Comment:
`const` has no effect here so it's encouraged to leave it out. (see [go/totw/109](https://goto.google.com/totw/109))

Reply:
Can we create some basic tests that passes in a non-zero `num_resource_variables` from Python? LGTM otherwise.

Reply:
> `const` has no effect here so it's encouraged to leave it out. (see [go/totw/109](https://goto.google.com/totw/109))

If I am not misunderstanding from the go link, this should be fine because it is for the definition of the function, so it will still provide functionality in that the variable will be a constant in scope of the function definition. I should however, remove it from interpreter_wrapper.h line 29, as that is the function declaration in which case it is unnecessary?

Reply:
> Can we create some basic tests that passes in a non-zero `num_resource_variables` from Python? LGTM otherwise.

Added a simple additional test in the interpreter to make sure nothing breaks with calling with MRV parameters. Also, by using the test for verifying fromBytes and fromFile are equal, it ensures that change to use allocator and MRV still provides correct execution.